### DISCUSSION TITLE
Redundant console.log statements in http.js

### DISCUSSION THREAD

**Discussion by oeyoews** [TiddlyWiki/TiddlyWiki5#8667] [General]
![image](https://github.com/user-attachments/assets/8437e360-0a2f-47e9-b3f7-f8b280b1036b)
I noticed that http.js has some console.log code, is this intentional? @Jermolene

**Comment by Jermolene** [comment]
Hi @oeyoews no, we should probably remove those. I think it would be good to get to the position where all of the visible logging that takes place on the client or the server needs to be explicitly enabled.
